packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -25,6 +25,39 @@ export class FauxClassGenerator {
   private STANDARDOBJECTS_DIR = 'standardObjects';
   private CUSTOMOBJECTS_DIR = 'customObjects';
 
+  // the empty string is used to represent the need for a special case
+  // usually multiple fields with specialized names
+  private static typeMapping: Map<string, string> = new Map([
+    ['string', 'String'],
+    ['double', 'Decimal'],
+    ['reference', ''],
+    ['boolean', 'Boolean'],
+    ['currency', 'Currency'],
+    ['date', 'Date'],
+    ['datetime', 'Datetime'],
+    ['email', 'Email'],
```

Comment:
Email should be type String

Reply:
This one is interesting..    Email is not a built-in type in Apex, but it is in the Standard Apex Library.  I was following the pattern used by Illuminated Cloud which maps this to a reference to the Email class....    Not totally sure which is the right approach...